### PR TITLE
EREGCSC-1743 Remove-Experimental failing

### PR DESCRIPTION
**Pull request by thwalker6**
Resolves # EREGCSC-1743

**Description-**

The removal step for the application is failing due to changesets.  Removing changesets fixing the issues

**This pull request changes...**

- Changesets are removed for automatically happening through AWS
- Remove-experimental works again

**Steps to manually verify this change...**

1. Open a PR
2. Close a PR after it finishes deployment steps
3. Close the PR.  Remove should be successful.

### CODE REVIEW

**Review by peggles2 — CHANGES_REQUESTED** [review]
looks good just one question.

File: solution/backend/package-lock.json [hunk 1]
```diff
@@ -2386,20 +2385,6 @@
         "node": ">=6"
       }
     },
-    "node_modules/fsevents": {
-      "version": "2.3.2",
-      "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-2.3.2.tgz",
-      "integrity": "sha512-xiqMQR4xAeHTuB9uWm+fFRcIOgKBMiOBP+eXiyT7jsgVCq1bkVygt00oASowB7EdtpOHaaPgKt812P9ab+DDKA==",
```
Comment (peggles2) [inline solution/backend/package-lock.json:2392]: what are fsevents? Is this related to changesets?

Comment (thwalker6) [reply solution/backend/package-lock.json:2392]: Thats an error thing for node nothing to do with this.

**Review by peggles2 — APPROVED** [review]
